sourcegraph/jetbrains
# Add CodyInlineCompletionProvider as an alternative for inlay completions

Fixes https://linear.app/sourcegraph/issue/CODY-3742.



# Based on PR #2303

## Full chain of PRs as of 2024-09-16

* PR #2304:
  `mkondratek/feat/completion-provider` ➔ `mkondratek/chore/ui-fixes`
* PR #2303:
  `mkondratek/chore/ui-fixes` ➔ `main`





## Test plan

AUTOMATIC
1. Run IDE in Remote Development mode 
2. Open a project, open a file
3. Hit Enter to trigger autocomplete 
Expected: autocompletion suggested

INVOKE
1. `shift + option + \` on **macOS**
Expected: autocompletion suggested
